Remotion/Data/DomainObjects.UnitTests/IntegrationTests/Synchronization/RelationInconsistenciesTestBase.cs

```diff
@@ -118,12 +131,13 @@ protected IndustrialSector CreateIndustrialSectorInDatabaseAndLoad ()
       return objectID.GetObject<IndustrialSector>();
     }
 
-    protected T CreateObjectInDatabaseAndLoad<T> () where T : DomainObject
+    protected T CreateObjectInDatabaseAndLoad<T> ([CanBeNull] Action<T> initialize = null) where T : DomainObject
```

Comment:
The CanBeNull feels like it should be redundant? It's more a curiosity though, than a change-this.

Reply:
I tried to make the parameter nullable, but nullable reference types are disabled in the project, so it is `[CanBeNull]` instead. Rider does not mark it as redundant, even though it is clear that a `null` default must mean that the parameter can also be `null`. My guess is that Rider/Resharper does not infer `[CanBeNull]` from the presence of a default value.